### ISSUE TITLE
Add linger parameter to `.close()` and `.close_all()`

### ISSUE THREAD

**Issue by ocaballeror** [opensistemas-hub/osbrain#255]
As mentioned in https://github.com/opensistemas-hub/osbrain/pull/252#issuecomment-368051072

This is mostly for Windows compatibility, since it won't support changing the linger value in the global config dictionary.

By the way, this also needs tests for both methods.

**Comment by Peque** [comment]
Yeah, let us not work on this issue for now. Maybe we can fix the global configuration problem instead.

**Comment by Peque** [comment]
There is #256 now, so i am moving this to milestone 0.8.0.

Note that this change would probably require not only adding more tests, but also allowing to pass a `linger` parameter to other method calls such as `.shutdown()` (and more tests, of course). So I am not sure it is worth it.

Anyway, let us discuss this for 0.8.0 instead. :stuck_out_tongue_winking_eye:
